Binance.Net/Objects/BinanceFuturesClientOptions.cs

```diff
@@ -0,0 +1,108 @@
+using System;
+using CryptoExchange.Net.Objects;
+
+namespace Binance.Net.Objects
+{
+    /// <summary>
+    /// Options for the binance client
+    /// </summary>
+    public class BinanceFuturesClientOptions : RestClientOptions
+    {
+        /// <summary>
+        /// Whether or not to automatically sync the local time with the server time
+        /// </summary>
+        public bool AutoTimestamp { get; set; } = true;
+
+        /// <summary>
+        /// Interval for refreshing the auto timestamp calculation
+        /// </summary>
+        public TimeSpan AutoTimestampRecalculationInterval { get; set; } = TimeSpan.FromHours(3);
+
+        /// <summary>
+        /// A manual offset for the timestamp. Should only be used if AutoTimestamp and regular time synchronization on the OS is not reliable enough
+        /// </summary>
+        public TimeSpan TimestampOffset { get; set; } = TimeSpan.Zero;
+
+        /// <summary>
+        /// Whether to check the trade rules when placing new orders and what to do if the trade isn't valid
+        /// </summary>
+        public TradeRulesBehaviour TradeRulesBehaviour { get; set; } = TradeRulesBehaviour.None;
+        /// <summary>
+        /// How often the trade rules should be updated. Only used when TradeRulesBehaviour is not None
+        /// </summary>
+        public TimeSpan TradeRulesUpdateInterval { get; set; } = TimeSpan.FromMinutes(60);
+
+        /// <summary>
+        /// The default receive window for requests
+        /// </summary>
+        public TimeSpan ReceiveWindow { get; set; } = TimeSpan.FromSeconds(5);
+
+        /// <summary>
+        /// ctor
+        /// </summary>
+        public BinanceFuturesClientOptions(): base("https://fapi.binance.com")
```

Comment:
Could you add an overload to specify a custom base address? Same for the socket client options